Launcher/CMakeLists.txt

```diff
@@ -19,10 +19,10 @@ add_executable(launch_alice WIN32
 	"${PROJECT_SOURCE_DIR}/src/map/map_data_loading.cpp"
 	"${PROJECT_SOURCE_DIR}/src/map/map_borders.cpp"
 	"${PROJECT_SOURCE_DIR}/src/map/map.cpp"
-	"${PROJECT_SOURCE_DIR}/src/map/map_modes.cpp"
-	"${PROJECT_SOURCE_DIR}/src/alice.rc"
```

Comment:
this seems like it would make the resulting exe have no icon

Reply:
fixed